### ISSUE TITLE
master build fails on Mac

### ISSUE THREAD

**Issue by songhappy** [intel-analytics/analytics-zoo#3254]
[ERROR] Failed to execute goal on project analytics-zoo-bigdl_0.12.1-spark_2.4.3: Could not resolve dependencies for project com.intel.analytics.zoo:analytics-zoo-bigdl_0.12.1-spark_2.4.3:jar:0.10.0-SNAPSHOT: The following artifacts could not be resolved: ml.dmlc:xgboost4j_2.11:jar:1.2.0-SNAPSHOT, ml.dmlc:xgboost4j-spark_2.11:jar:1.2.0-SNAPSHOT: Failure to find ml.dmlc:xgboost4j_2.11:jar:1.2.0-SNAPSHOT in https://s3-us-west-2.amazonaws.com/xgboost-maven-repo/snapshot/ was cached in the local repository, resolution will not be reattempted until the update interval of XGBoost4J Snapshot Repo has elapsed or updates are forced -> [Help 1]
[ERROR]

**Comment by qiyuangong** [comment]
I think this building issue is related to xgboost-SNAPSHOT discussed this morning. @dding3

**Comment by dding3** [comment]
@Le-Zheng Do we have xgboost binary for mac version on our Nexus mirror?

**Comment by Le-Zheng** [comment]
You can add this maven mirror:
```
<mirror>
      <id>ardaNexus</id>
      <mirrorOf>*</mirrorOf>
      <name>Arda's nexus</name>
      <url>http://10.239.45.10:8081/repository/maven-public/</url>
    </mirror>
```

**Comment by songhappy** [comment]
still failing.
Could not resolve dependencies for project com.intel.analytics.zoo:analytics-zoo-bigdl_0.12.1-spark_2.4.3:jar:0.10.0-SNAPSHOT: The following artifacts could not be resolved: ml.dmlc:xgboost4j_2.11:jar:1.2.0-SNAPSHOT, ml.dmlc:xgboost4j-spark_2.11:jar:1.2.0-SNAPSHOT: Failure to find ml.dmlc:xgboost4j_2.11:jar:1.2.0-SNAPSHOT in https://s3-us-west-2.amazonaws.com/xgboost-maven-repo/snapshot/ was cached in the local repository, resolution will not be reattempted until the update interval of XGBoost4J Snapshot Repo has elapsed or updates are forced -> [Help 1]

**Comment by dding3** [comment]
I think after we add the maven mirror @Le-Zheng mentioned, it will download the jar from the provided mirror other than s3, did you remove the "XGBoost4J Snapshot Repo" mirror in the pom?

**Comment by songhappy** [comment]
fixed in #3239
